mozetl/taar/taar_lite_guidguid.py

```diff
@@ -0,0 +1,189 @@
+"""
+This ETL job computes the co-installation occurrence of white-listed
+Firefox webextensions for a sample of the longitudinal telemetry dataset.
+"""
+
+import click            # noqa
+import boto3            # noqa
+import datetime as dt   # noqa
+import json             # noqa
+import logging          # noqa
+from botocore.exceptions import ClientError   # noqa
+from pyspark.sql import Row, SparkSession     # noqa
+from pyspark.sql.functions import col, collect_list, explode, udf, sum as sum_, max as max_, first  # noqa
+from pyspark.sql.types import StructType, StructField, StringType, LongType, IntegerType  # noqa
+from taar_utils import store_json_to_s3
+from taar_utils import load_amo_external_whitelist
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+OUTPUT_BUCKET = 'telemetry-parquet'
+OUTPUT_BUCKET = 'telemetry-parquet'
+OUTPUT_PREFIX = 'taar/lite/'
+OUTPUT_BASE_FILENAME = 'guid_coinstallation'
+
+AMO_DUMP_BUCKET = 'telemetry-parquet'
```

Comment:
These variables are unused.

Reply:
These are used by the click decorators in main()

Reply:
I only see `OUTPUT_BASE_FILENAME`, `OUTPUT_BUCKET`, and `OUTPUT_PREFIX` in use here.